enterprise-cfengine-guide/federated-reporting.markdown

```diff
@@ -361,11 +361,13 @@ $ cf-remote -H $CLOUD_USER$SUPERHUB,$CLOUD_USER$FEEDER sudo "/var/cfengine/bin/c
 
 Uninstalling Federated Reporting from a superhub is not possible at this time.
 
-In order to remove Federated Reporting from a feeder simple do the following as root:
+In order to remove Federated Reporting from a feeder remove the ```cftransport``` user
+and ```/opt/cfengine/federation``` directories. This will remove trust established
+with the superhub and will prevent superhub from performing any dump/import procedures.
 
 ```
```

Comment:
console after tripple tick?

Reply:
After all the console blocks? I didn't notice a difference in github, maybe in generated docs?